README.md

```diff
@@ -26,6 +26,8 @@ Clean options
   --stripEmptyTags  Removes empty tags such as "defs" or "g".
   --stripTags  Removes tags, that are listed in this list.
   --stripAttrs  Removes attributes, that are listed in this list.
+  				Should be repeated for every item, for example:
+					svg2sprite <source-directory> <dist-file> --stripAttrs id --stripAttrs viewBox --stripAttrs x  --stripAttrs y --stripAttrs width --stripAttrs height
```

Comment:
You can add this changes to CLI help command?

https://github.com/mrmlnc/svg2sprite-cli/blob/master/src/cli.ts#L19-L38

Reply:
Yeah sure, later today I'll send the changes in the PR.